build/rust/Cargo.lock

```diff
@@ -264,6 +264,15 @@ dependencies = [
  "subtle",
 ]
 
+[[package]]
+name = "brave-ads-cxx"
```

Comment:
`-cxx` is not necessary here, but also `brave-ads` isn't really very useful as a name. The purpose of this crate is to sanitize html so it should be named along those lines